### DISCUSSION TITLE
"Update to GCC copyright assignment policy"

### DISCUSSION THREAD

**Discussion by shenlebantongying** [rms-support-letter/rms-support-letter.github.io#7103] [General]
https://gcc.gnu.org/pipermail/gcc/2021-June/236182.html

Thread view

https://gcc.gnu.org/pipermail/gcc/2021-June/thread.html#236182

**Comment by 6r1d** [comment]
Yet another wave of absurd changes.

As far as I understand the current situation, most of the [GCC steering commitee](https://gcc.gnu.org/steering.html) members are now related to some corporation. (Yes, I've read GCC mailing lists where people are telling "there's the most talent in corporations" in denial of the corporate interests influencing current events.)
In the context of what Leah Rowe [wrote](https://libreboot.org/news/rms.html), to me it looks concerning.
There probably is an attempt to reduce FSF role and increase role of corporations gradually.

**I wonder if there a ray of hope** with this change preventing a switch from GPL, as [stated here](https://gcc.gnu.org/pipermail/gcc/2021-June/236219.html)?

**I've also heard** that it'll become harder for FSF to enforce GPL license if trick above is done. How true is that?

**Reply by weimzh** [reply]
historically the RMS/FSF's version of GCC ('GNU C Compiler') was forked into EGCS, and EGCS replaced FSF's version later because it's technically superior and the version developed by FSF died. GCC steering commitee is basically the people who led the EGCS fork which later became the 'GNU Compiler Collection' aka. the modern GCC.

If the FSF is not the copyright holder of a piece of code, they have no right to enforce the license. Only the copyright holder has the right to enforce the license.

The FSF will not be able to change the license from GPL to something else either. The good side of this is if some evil company takes over the FSF they won't be able to relicense the code either, and the license can still be enforced by the respective copyright holders if the FSF goes bankrupt.

----
EDIT: I'm just trying to clarify some facts and I'm not interested in whatever arguments
